server.js

```diff
@@ -1,30 +1,109 @@
 import express from "express";
 import cors from "cors";
+import booksData from "./data/books.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+// Defines the port the app will run on, can be overridden 
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
+// Middlewares to enable cors and json body parsing (should not be changed)
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+
+// ROUTES
+
+//Startpage with definition of routes
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  
+  res.send({
+    "Books API" : "Look for books by ID, ISBN number or title",
+    "Routes":[{
+      "/booksData": "Get all books",
+      "/booksData/isbn/:isbn": "Get a book by its ISBN number",
+      "/booksData/id/:id": "Get a book by its ID number in the database",
+      "/booksData/authors/:authors": "Get a book by its author/authors",
+    }]
+  });
+ 
+});
+
+
+//The first request to get the whole array of books
+app.get('/booksData', (req, res) => {
+  res.status(200).json({
+    data: booksData,
+    sucess: true,
+  });
+});
+
+//Find books by their ID number of the array by .find() method
+app.get('/booksData/id/:id', (req, res) => {
+const { id } = req.params;
+
+const bookById = booksData.find((booksData) => booksData.bookID == id); 
+
+//Deliver a status of 404 if the ID number is not found and 200 if it returns ok
+  if (!bookById) {
+  res.status(404).json({
+   data:"Not found",
+   sucess: false,
+  });
+  } else {
+  res.status(200).json({
+    data: bookById,
+    sucess: true,
+ });
+};
+});
+
+
+//Find books by their ISBN number by .find() method
+app.get('/booksData/isbn/:isbn', (req, res) => {
+  const { isbn } = req.params;
+    
+  const bookByIsbn = booksData.find((booksData) => booksData.isbn == isbn); 
+    
+  //Deliver a status of 404 if the ISBN number is not found and 200 if it returns ok
+    if (!bookByIsbn) {
+    res.status(404).json({
+     data:"Not found",
+     sucess: false,
+    });
+    } else {
+    res.status(200).json({
+      data: bookByIsbn,
+      sucess: true,
+   });
+  };
+  });
+
+//Get the book by title by filter.() method
+app.get('/booksData/title/:title', (req, res) => {
+  const { title } = req.params;
+
+  const bookByTitle = booksData.filter((booksData) => booksData.title.toLowerCase() === title.toLowerCase()
```

Comment:
Nice work to use both filter method and includes method and convert to lower letters.